cache/statement.c

```diff
@@ -0,0 +1,32 @@
+#include "statement.h"
+
+int redis_del(redisContext* c, const char* const key) {
+  redisReply* reply = redisCommand(c, "DEL %s", key);
+  if (reply->integer) {
+    freeReplyObject(reply);
+    return 0;
```

Comment:
Do you consider using redis status code?

Reply:
All reply types are non zero. Do you think it's still good to return this way?

Reply:
@wusyong 
Got it. Then let's keep it this way, maybe later we can design our own status code for whole project. :smile: